1.0/docs/devguide/templates.md

```diff
@@ -134,8 +134,10 @@ model data that generated a given element. (There are also corresponding
 To filter or sort the _displayed_ items in your list, specify a `filter` or 
 `sort` property on the `dom-repeat` (or both):
 
-*   `filter`. Specifies a filter callback function following the standard 
-    `Array` [`filter`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/filter) API.
+*   `filter`. Specifies a filter callback function, that takes a single argument
+    (the item) and returns true to display the item, false to omit it.
+    (Note that this is **similar** to the standard 
+    `Array` [`filter`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/filter) API, but the callback only takes a single argument.)
```

Comment:
Could make a little more clear about what's different ala "...but the callback only takes a single `element` argument, and no `index` argument, for performance reasons."